### PR TITLE
bump lightning-terminal for wasm-client & mobile

### PR DESCRIPTION
**Pull request by ViktorTigerstrom**
This PR bumps the lightning-terminal version for the wasm-client and mobile to `v0.10.5-alpha`.

Happy to address any feedback :)!

### CODE REVIEW

**Review by jamaljsr — APPROVED** [review]
tACK LGTM 🚀

**Comment by jbrill** [comment]
LGTM!

**Review by jbrill — APPROVED** [review]

File: cmd/wasm-client/go.mod [hunk 1]
```diff
@@ -6,19 +6,18 @@ require (
 	github.com/jessevdk/go-flags v1.4.0
 	github.com/lightninglabs/faraday v0.2.11-alpha
 	github.com/lightninglabs/lightning-node-connect v0.2.3-alpha
-	github.com/lightninglabs/lightning-terminal v0.10.1-alpha.0.20230612093826-ee04747e4a34
-	github.com/lightninglabs/loop v0.24.1-beta
+	github.com/lightninglabs/lightning-terminal v0.10.5-alpha
```
Comment (guggero) [inline cmd/wasm-client/go.mod:9]: Should we bump the version of `lightning-node-connect` to the latest tag (`v0.2.4-alpha`) as well? We do replace the dependency further down, but that way we at least keep it recent.

**Review by guggero — APPROVED** [review]

**Comment by ViktorTigerstrom** [comment]
Thanks for the reviews @jbrill & @guggero 🎉!!

File: cmd/wasm-client/go.mod (hunk 1 above)
Comment (ViktorTigerstrom) [reply cmd/wasm-client/go.mod:9]: Good point! Addressed with the latest push :)!

**Pull request merged**